exotations/solvers/exotica_levenberg_marquardt_solver/package.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>exotica_levenberg_marquardt_solver</name>
+  <version>1.0.1</version>
```

Comment:
Are you okay if we change this to `5.0` before we run our binary release to melodic? Helps every package in this repository be at the same version level

Reply:
I wouldn't mind. Does every package of the exotica meta-package need to have the same version number?